front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.scss

```diff
@@ -6,3 +6,29 @@
 :host ::ng-deep .p-disabled > .p-radiobutton {
   opacity: 0.65;
 }
+
+#form {
+  max-width: 360px;
+  width: 360px;
+}
+
+@media (min-width: 576px) {
+  #form {
+    max-width: 540px;
+    width: 540px;
+  }
+}
+
+@media (min-width: 768px) {
+  #form {
+    max-width: 720px;
+    width: 720px;
+  }
+}
+
+@media (min-width: 992px) {
+  #form {
+    max-width: 780.4px;
```

Comment:
Thanks @Elaine-Krauss-TCG! 

Should this be `780.4px` to match @sasha-dresden's suggestion, or is the extra `.4` on purpose?

Reply:
I put it at 780.4px, since that was the exact maximum width that I had observed prior to these changes.  I could reduce the width of my browser, and the form would stay at 780.4px until I crossed a threshold (I did not measure what the threshold was).